hbase_regionserver/metadata.csv

```diff
@@ -7,9 +7,9 @@ hbase.regionserver.ipc.total_call_time.mean,gauge,"",millisecond,"","total call
 hbase.regionserver.ipc.total_call_time.median,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.ipc.total_call_time.percentile.99,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.regions.num_regions,gauge,"","","",Number of regions in the metrics system,0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_ops,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_batches,gauge,"","","","",0,hbase_regionserver,""
+hbase.regionserver.replication.sink.applied_ops,gauge,"","","",Number of WAL entries applied on replication sink.,0,hbase_regionserver,""
+hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"",millisecond,"",Replication time lag of last applied WAL entry between source and sink.,0,hbase_regionserver,""
```

Comment:
would you mind adding `unit_name: millisecond` in `_metrics.yml`?  `_gen.rb` supports `unit_name` in `_metrics.yml`.